pkg/providers/cloudflare/helper.go

```diff
@@ -117,6 +117,34 @@ func (p *CloudflareProvider) addAccountMember(ctx context.Context, email string)
 	return nil
 }
 
+// removeAccountMember removes a member from the account of the provider
+func (p *CloudflareProvider) removeAccountMember(ctx context.Context, email string) error {
+	ctx, span := startSpan(ctx, "removeAccountMember")
+	span.SetAttributes(
+		attribute.String("peer.service", "cloudflare"),
+		attribute.String("span.kind", "client"),
+	)
+	defer span.End()
+
+	member, err := p.findMember(ctx, email)
+	if err != nil {
+		// If the member is not found, nothing else to do
+		if errors.Is(err, errMemberNotFound) {
+			return nil
+		}
+		return err
+	}
+
+	_, err = p.client.Accounts.Members.Delete(ctx, member.ID, accounts.MemberDeleteParams{
+		AccountID: cloudflare.F(p.accountID),
+	})
+	if err != nil {
+		return fmt.Errorf("failed to delete member %s from account %s: %w", email, p.accountID, err)
```

Comment:
we could decide to not return an error here and just log the error.
the main goal of the revoke is to remove from the group, which is already done as this point.